docs/api-reference/next/head.md

```diff
@@ -61,3 +61,5 @@ In this case only the second `<meta property="og:title" />` is rendered. `meta`
 
 `title`, `meta` or any other elements (e.g. `script`) need to be contained as **direct** children of the `Head` element,
 or wrapped into maximum one level of `<React.Fragment>` or arrays—otherwise the tags won't be correctly picked up on client-side navigations.
+
+> We recommend using [next/script](https://nextjs.org/docs/basic-features/script) in your component instead of manually creating a `<script>` in `next/head`.
```

Comment:
```suggestion
> We recommend using [next/script](/docs/basic-features/script.md) in your component instead of manually creating a `<script>` in `next/head`.
```